53. Как выполняется обработка исключений в Python?

Для обработки исключений Python предоставляет конструкцию из трех слов: try, except и finally. Синтаксис выглядит
примерно так:

try:

# попробовать сделать это

except:

# если блок try не сработал, попробовать это

finally:

# всегда делать это

Ниже упрощенный пример такой конструкции. Здесь блок try терпит неудачу, поскольку мы не можем складывать целые числа со
строками. Блок except устанавливает val = 10, а затем блок finally выводит complete:

try:
val = 1 + 'A' except:
val = 10 finally:
print('complete')

print(val)

# => complete

# => 10